### ISSUE TITLE
lint MARK comment formatting

### ISSUE THREAD

**Issue by brennantaylor** [realm/SwiftLint#749]
For example enforcing and autocorrecting.

```swift
//MARK:bad
// MARK: good
```

**Comment by jpsim** [comment]
good idea

**Comment by freak4pc** [comment]
I wouldn't mind tackling this if I have some spare time tomorrow.

**Comment by krodak** [comment]
Just added a relevant PR for this

**Comment by jpsim** [comment]
Done in #768.

**Comment by brennantaylor** [comment]
THANKS!

**Comment by Igor-Palaguta** [comment]
Should be same rule applied for TODO comment?

Not clear why for MARK it is required. Xcode works properly even if mark without leading space
